### DISCUSSION TITLE
Custom word pronunciations in XTTS

### DISCUSSION THREAD

**Discussion by madey83** [idiap/coqui-ai-TTS#195] [Q&A] [answered]
Hello,

is there any place where i can define how sound should be genereted for some specific words, for instance "Poppy" or "km"  in polish it sound strange and not accuratetly..

**Comment by eginhard** [comment, marked as answer]
You could fine-tune on data that includes these words, but there's no way to force a specific pronunciation. You can use a text normaliser for tokens like "km".

**Reply by madey83** [reply]
> You could fine-tune on data that includes these words, but there's no way to force a specific pronunciation. You can use a text normaliser for tokens like "km".

thanks for reply.... 


i can't do fine-tuning for each made-up name... "Poppy" is english made-up name... therefor is for me better to define some variable and put inside how it should be spoken in polish.

how can i do it: "You can use a text normaliser for tokens like "km"." can you give some example?

**Reply by eginhard** [reply]
A text normaliser would convert "km" into "kilometer" - you'll need to check if you can find one for Polish.

**Comment by madey83** [comment]
A text normaliser  is file in Coqui or what?

**Comment by madey83** [comment]
n/a
